server/api/zippedBoilerPlate.js

```diff
@@ -1,72 +1,85 @@
 const zippedBoilerPlate = require('express').Router();
+const { models: { Code }} = require('../db')
 const ejs = require('ejs')
 const fs = require('fs')
 const zip = new require('node-zip')()
 const path = require('path')
 
 //helper function designed to deal with multiple files in a directory folder
 function zipHelper(fileString, arrObject){
```

Comment:
zipHelper should be async, and all zipHelper calls should have await in front of them